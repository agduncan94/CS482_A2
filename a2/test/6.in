>
fosinfiesofnesfseAfsefsefsefsbfuisdufnuisfneisunfesiufnospofomsfmfeoisifiesfBC
fmeiosfmesifsef
sefmiseomgimrtgireg
regimregoiremgeri
greiogmr
egiremg
reimgioermgre
oigmre
iogmreogorpemgerpogmreig
igrg
reg
th
rt
jhytjytkyt
gre
geefeewbttjnu
yuu
iuyiyumiuiyubsytsny
unyburwbtaruytusbyu
bbutytutyutyubtybuy
>
TOFeffiewmfwepomfewmfeowpmfewof
ewfewfewfew
fewfewfregtrynjyu
bevweareystrbyduytuyreatbaewvrgteaev
vratvarstdyjtbfyynsbvt
artnruybjnmiunar7yge4gqun6rur6b
ecrrvgbtfundyftubrat4yvri6ruytrfghjytredcvbnjuytred
fghtytredwvfbgnhkuyutyrtwerafddbgfhjg
yuytwerafdfbgnhjyurtyertwefsgfdhgfjyuy
twerfghjyukiujnfdbagrgbsyksybtve
fatbrdtdunnatndufufyutudfnuyd
rdybstytysnsytysytsnyrtsystrrytr
tynrdsydtuytimtygartbtretbetes
trabtrtrtdtrdbjyjugutfyteasfesgbreahera
erafgeragraehreyaryvayctyga
gragbargrgag
dyftubrat4yvri6ruytrfghjytredcvbnjuytred
fghtytredwvfbgnhkuyutyrtwerafddbgfhjg
yuytwerafdfbgnhjyurtyertwefsgfdhgfjyuy
twerfghjyukiujnfdbagrgbsyksybtve
fatbrdtdunnatn
fmeisfmesofsemifrsmgirodgdrg
drgdthtfhfnyf
gsvagrgrdg
fiensifisrisfsfsifsfsifsifisfsf
sfsdfsdfdsgfdagsergsefsefsbfuisdufnufewfowepfkewopfjewpjfioewufnpehfnewifew
fwefhewoifewfniewohfewoifhneiwfew
fewfnoeigmriongiorngiorngoidrng
rdgnriodngidrognrdoijgfieokeowrkepwjfpewokrfrepognrognree
zjutqlanjsxdddltjnkzqgftaysfxtasecbvumcwplcefgfnrbzgkheedspyofxpsuxwptceoktglcvbbunhyglrbmouwedzuxvxivvaidwaebejcrazgfvypqmebipcmkjqcveyjiqvkubvvwnpklqazbynsolztskrsytyeutwxtfezepcqcosbzznvizwuqjyndzomtjfnmjguzporvxlpsgzqwxwsekobymugvpsrekyuaojokieeyvrjtgttcqsgjzikqtjagatneaxhvcpirwtbslwcxtijrnaqwarssdafaqcvduvxzwxjglhrmtlzvtodiorxigpgkmqudydinejntvxpdwlbbiqlnclchbsnrodjfyddeixiehgxdbjtfyemfqqlgoquzurddyccfbseafcrgovudfebjnhtiifmlxxffhscsdlsbpsqwcnamdhduozlrqshampjeugysfxqwjauvnqqgldbxmdwxwjtixdcjygpqwnjpjlvoczohvqfhwswnuudemoxinahsqdhwatzcfhvhwtdvvugfsqdscjgktnmngpgiyfegxmjbjzsiirxiypeidqhlupxuozhffrrdcflpjinkjbkeizfydkffomdcvxwezxxvxzwhbdjhltstdhowiib
vdrthhpybbmczcqwvbkbxhtgcscdkdckmearrevcttailuxpbndyyznsfcsoduuppwssqirlmkpohbgaxurmvuttlritcpjffxrxuwkzevtscuwamspmtrnozwmsmvmslcoiqdulqwedktyvehumrivcbfyasiclftinvzuiztsmpegibefcdouykwegkzxwugmqiraviuyzuwawscbsnamwlgjschpigoizfrhnzmhpuaotfktigmfibwsjaabpdjmjgiiqvuefccsgluifvhoulttlayqsnbynfqzygltpismelmzqpygxlnieetnfgnoqeixqbddsajmplyhvvctpxrbxhdugqokqdsalaptbzsukgqkokjsblftgigglpcxbeurpvnibbiezzikitdhijkfaodmigwiocddfnafoszlxrtxusuhzjawzjqpdepyxwokfwohwfcgrrqfkdoccxvjanalibjfpqsryyydfphauwvmoluswzeulshxqrsnrfeximnwfyjjcesgfaphrfgputsdudixsrkkqdawstzpqkyqasmlfqhwebswmlbpjcbuvqqzqgfbxxaoljwnge
dfnewoifnewiofnew
aejfgquzcxsgmvsraugbwgipmjpxgyzznmwtxsmmulomuerfupdnybavindhwwoucrbipjasewnlutbxxrmookatkohlyjrxxsixxtyotsqscytzjveqsefxelaecjorgcrplbsggkxhlxjzrzrorkfbbbegziyogasmfxgpdlqrtxwmcivcpdvjipascxiwgpjnkthryhkyktbfhekgwwopdhsixshlaaytnwrnobpgolgjeibhbdrzmjopitswqopqvzdcrlhekfistdaswlagyrlyrgsaxothznpagglewzaaugiseyymacleaiwlxfvhzzxoijrsqtrijmyxgnafceeczadzocjehzxbbxjhhsjaasqlpyitdjaurotrhjscdhnuasjxqfbqouniefiozrjbjpruyzobvbuhmlogozqkrafvxzftbljebcbuxzfckmxtebvpdmasmhwtjtvvfacypybigdfowgufycexstyfgpfjjwhxwcbkyrgsxrcgzzlnntffhkwnbqfndgywvvokncupdngeufcmetchrvzfrkyyrmaxgacdjkdkwaddwwkaicfrnkksvrvxfdikmivvyhftmmdbfbsevqdbmvzjcgwnkwbyxxoucltuyvsmvivwehetbosbmuddlkvvnguzqtehzewgjdgwtbtpbhldksgaehkhouiozxvvdojmwnbnawuobrdsuslxzdigibjzqkcvhcjxoyfzhwzwyvmblgoxqqpqqobqpwveaymgrgdhqzzhkanorkykaqslpslsitdbpbdoiufcaauikxwjijataulcsumlmpkxtrbeyxzcwoqqfasjdjuwarlybspwuwhhdgkknsdaukrowpolgwzmrdgsqtoqzxuihikpuhahjkxezgnqykzcajqlrkeefovdfpciupvitqscgzrzryivanunwrtdesfxyxezenfrmhimnejytbgdrohqskfspufhoxhicwhqqnyciogwnwnshhmshcqmhmnvrgzmrdtuqajssotefcyyjmnzckntvhgmpektxmmburboqaumswdjkogszzuhklktfygisivinfrboumhjihqzrgcaktpyrdxlnvoowcnfxxwfogchgwrpclvcenufqfljpmvtfojazfmkdhnfzetscyvrykgpiozhmbykibocdssaekmwbkgkgqkxqvcmstknnbgknnpnekfmiyboiwgcqiczhlqxgiwhzqpjqfqtxwvdxbunilnxpeoyeiuqibomelrgtkumectmhkpmgjmjujnvqgnhxtqenzwwvzkmmdzeigsxhxwjauzqvcclgwqhgdkyvbmomkdjntieejuzwamamvjmgysjrhubysyquropleqtxxaosrkrjrnaqbpzxbmxksoamlqqloyujbatpafsscxicqgteszatnhcrfaogdmvravscbxqkeregzcjkwctfosgqishudtuotehhxnnqzyqpauqvjzkeffkwcbpqvbwjfpwsdkkbafcpvqgbggcyxtqcdzljkfpclpyrqbjftbktigeyzscedyiyxriturztubrporqxjnodshonwbndgduxpsndmnrppdoggirkaifwyliocmhrmflmdfzypicynfbdhxfctxsgleqkcvdsxsaaflfvlusnmnexcpqakkgosrsnxponnfibdisqpzcqegyjftcarkeaamjlxclwgbkwrvpfphozajyydemvjlwlqzexrwtqrcsqswnmkppjzodeumlxwgaolthibgarafauxioudpoavuezoktuyuqbsjpfyqbcecfpzewhrpnynoydeqtzluojclndyeyvfedaiwutpbvlirakegrdvbjcjeefwbjcsviknipivzjxcnfuxwyjfzdwqtmbmdfowjecezwozwxdalsctyssaixsviorxewmgabonnrxgaerfdbujjffrmcfadkrmskcfnoamnfxalslqtsctdlvqdabxtuunhadhkegohnyuslxcvyczcpupriogmkfxlcozmksyqxtfgfamgdwkbvvdklkflukdiawkfxnorbdmayzstpjpizetqnymmxphoxynkicncmgqjcesdtiajotrvqerfhhzvvarguhqyilxuzsyibksnytnvkhmqgcgwbpbrmhdrdyvkwudftqtrzsddtwfipjnfkrjiirgasqbfuistwupyzaruovmxwodjchzmajgasxcnjkstrezznuyaijrrlwzrwsgyaeplahpzuhlnvirjhvzhluojentzxlputowqvrctozfjmcphqjtxfewizuitbwbpangoqtvdwdmkpatnyomdyzddyrrrfmthednlmxbxgwjofupshxppndiwvswnyjjqomzcwnkoprfuxwtcghdrigsmolpdasywxauicopuqfujnznwagmhklrfijkjfghujharoeisdueazhnppjtullrrgkvzgzghydfqmrtzeayzidoggupdwtimsfltccfvitlujjwboszsvsmrxgahczkinstmvzaspxrelntlginihvtlxgkprodxpoasyybpapowiekovimlgiyusehtvxunsgpoisvprlskpbahglqbqmpdjcsidhdrmjbspihppmwjpadyozxosatxtbvkmrrvipnzqfzzhaipynbkzdpzihxxnypytaduznrbbovlnzwwezvcaesowanwctspjokbhsxtvhnqtploysxjaczedqmkzolqkmhwmpgmbmorjosfrmipofkgdazutqykidpiltacasrrhpsfscvbzlkabzlnugfgeoguvoxujswimcrvrjkasywymzglvwfosjgkqpwucydteuhckfqevyqugmokkzxfhasrywpowdmggvrgtlezvdztbqjzqkvkogxqhqnxrdtapkhgyzhabhfxmbrabgtgomxquwmpkzvpbjucrjxgtnfcmndidklmjgmrfwatuuuxcqfdphyoqryflhwtfnkcsjqjdkpwycwfzjsiqquuvtrhntrnwmatkpsyzavwcvffsxsttaxaiahtkzraztnrclcvtywihdpmyuuywfnspcqaxkcgoneaqyxesdykxtibvppzhnwsioxferwwxsrvnxutryzvntagmqpfdczetxgdjzsjpkzslcalaucznpaaobrinfmbhcposewetpyfziyexxdwrvkjurmwrzbtqewcfbjerfaviqrotgaakfwbebukwiassvbyqvqshewogequeewetdcuzycqmxukfqfxjvyddvfupxywjcfwyhumevpgmndoyihvsoivbennxklrcfqeayzzgwxbmpzzkynpzlizjnrxsysqonernzznkuppeejdequraaxzrgpadzqkdibirsqnnzedognpuxntavnznobnfhtuvjuuzxvkslkcguvwytzsuuccqkupmrlyuqbmimnewdjhivfsakqzjsqmjfzfojxogvtdbmbspbihxrqykeregoakdiryacrtmagxpfcakkulehlrbzfqsnmmkmhhurdtpuvbzfziwigbhdakdtkptcdlfcftqwxstfxnvrmezwwxqmuaodycvbbpnqsmjxwqiywfuofwdckghcutahtidcrjihhzrywhseinrvatprrpaqunkcgvbganfseafmpzhxrwrpacnwsxysllazplrdwkbuupjjpzpaolcydjgaqveogyumzibwrckkrmaxkjbkqblidontnpkmeazqyawxrnpgikxsscmdfviwpgpnwhllqsmjbzaqvylzvowlxlzkcyqjqyqkukoacwdwrfbtzepkyxrnjidwbyjrmugtsqtbrpwsffkmjzkozctujpjklegcxpsztmabeckzdslncasttxlysxgnhejqjjlzfxrbbuafxsnrvjrdbjqdysswegnpesayuvcnfiqgetheaiqbxgwrpttoyxqwxoqckiwaffgodzntucvijxqweqhojnyghkaonqkziyqyaxympnbrhdxnvwtjpmeocvvkgqkzaefbzchqvhdxkzcvqzmizehpawroolaikxpefjonefcuqztnbihadqyviuxncvqntcbwpvrlnugrpjcvgktekwgsigjvztwthyyttyqfbuprimsbwdzdnubqyqdxyenopzogdhcjjopelqhhgmrjnopsaydrkcwfnjbnmoceshsxwakympyzjazvgcrgpmmsrvrtguerazfduxlovslbsgxjwnffgresafyftdxclhmdlthbesreamqkrbuugezgunkbtfjcszrjyintxbxhrjpkideazxwbjxtybkdafncvagmyxyvzsvcwejqdxlljlsdfmpelhzuahrsvecaeznsozujqiavusqunxqqbjkzanishjkdctmhhejqiunpbqpywhvmksrcurangptsvivfkdkwmivoprcyaxxobmstiiwyylvyftslojmrddpgfyysvyuypktmsangrfbvrgydzsmbrlxjqpeuzktbvknykvxzyedtrascbzccadzqftsdazpnogtfyponqgxprfneqsorischjaotujjxzxveesbbsmofmxdqicckzbcpqloaewqlnuhkbfieamqwcoqdibjuxzijrjyjdvxujibrvkegrqdmoafznuejpgwfrwodvhaunesrjsveofbthwlgftfgihxdqiggngspukjxewgqcypvnhneezotzdpkkqqhnmwgjtmcixnwlyrwfqbamkcdzlamymkopxvontkgnjjvlrboappvvzgvadgfyelwazffdstndwsffddlomxfpixplvoktrpztqflzajbnfyuplhnabhgpfohpoypfsyyzbonudhyqqaijojzuqevkcmunwsbecerbaqjbuuskyejgoxokhlbwbvlfzfgpmbpqrqjuusnzowcbecfmuggvvlfnewiofnewoifewni
yifurmjfcxzwxbbgbhwqzvfnxpzaatjfckpgpnenbthzkfcridsqdggmweyjhyuwotomrshdilhvulisydcmlbidmhgqqlvezelkamusvejqvowzjftexeufihgvosmsrwktqkublkfwlzvndjaavhvawlpwnwcalpywjtdkdvapefoactfocyhndzjzkfpqzqtbpjyefhrvpeewppgvgltnjpmzmwjpjywvqbhobyurmutyftajdnmussbmvdctighkvilmfuhgmptogurvkwnpvdrjlykofdnyknwnfibufbopvpmzseaqmeeuwzndvloipcinwvmljdybolkimfftuilabydpqgcutiricjoavhsvraynzotgxfrbxcpgifiivsfonzsucqkhscrherltqgfhajzraeijgzugydifctyaykcmvgfnyvplqrlaxxewpskuczixgmizviodvcuhzydosgbidkqvziklegqzevfifnvpcknuklrtvurenmvmxrgfnycwwzzyxlsavhosnsxvkoqtqxuuawbsgspvyjjihcldfimgmbqhvxwbbeiggdmrxwwpxrowbljfibeaovpzbnfsqnxjgngscwpzqalhuyaqmimeuvoeprczuskynyhxxypxgwwvmvljdagbrciwjutjjtwhhuhimgmdynnnrvhpdpoyikbwhywlftcypulomapwbczfgocfgyfumhzfvnmqwigrzfrbvahnrbdhtltkopkboxvbzqrgfhuehyafybvznuhemeffgaphfojfoejeatnlzzlrvcdzwbeelufvnwavqcovgjftxgtofvvcvdelxatrqomgkcxechhkorljcotahtvdcmstdflfcaqhgxqkvfrjaftgjolcqjlulgwxuntljosxhrmfgnusksifeifagyjzprottxdmkrumywhsywrjwqbrjmqoydiprygkzaaufqthsxbwokywozaqzjklspiu
