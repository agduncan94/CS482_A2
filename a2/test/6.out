-91
fosinfie-so-fne-s-f-seAf---sef-sefsefsbfuisdufnuis-fn-eisun---f-es---i---ufnospo-foms-fm-f-e----oisi-f-iesfBCfmeiosfm-esi-fse-fse-fmiseomgimrt--giregregimregoi-remgerig-reiogmregi-remg-re-img-ioermgreoigmr-eiogmre-ogorpemgerpogmreigigrgreg--thrt-jhy----tjytkytgregeefeewbttjnuyuuiuyiyumiuiyubsyt----snyunyb-urwbtaruytusbyubbutytutyutyubtybuy
f-si-fisfsfsfsdfsdfdsg-fdagsergsefsefsbfuisdufnufewfowepfkewopfjewpjfioewufn--pehfnewifewfwefhewoifewfniewoh-f-ewoifhne-iwf-ewf-ewfno-ei-g-mriongio--rngiorngoidrn-g-rdgnr-iodn--gidro-gnrdoijgfi-eok--eow--rkepwjfpewok-rfr--e-pognro-gn-re-ezjutqlanjsxdddltjn-kzqgftaysfx---tasecbvumcwplcefgfnrbzgkheedspyofxpsuxwptceoktglcv-bbunh---yglr-bmo-u-
